onnxruntime/contrib_ops/cpu/tokenizer.cc

```diff
@@ -114,6 +136,25 @@ Tokenizer::Tokenizer(const OpKernelInfo& info) : OpKernel(info) {
   }
 }
 
+Status Tokenizer::EstimateNumberOfTokens(gsl::span<const std::string> input_span,
+                                         size_t& max_tokens_per_row, size_t& total_tokens_estimate) const {
+  total_tokens_estimate = 0;
+  max_tokens_per_row = 0;
+  for (const auto& s : input_span) {
```

Comment:
Should you include StartMarker and EndMarker if mark_ is true? Maybe it is added later.

Reply:
This is done later.